### PR TITLE
6lowpan nc: rtr sol timer expects an interface

### PR DESCRIPTION
**Pull request by OlegHahm**

### CODE REVIEW

File: sys/net/gnrc/network_layer/sixlowpan/nd/gnrc_sixlowpan_nd.c [hunk 1]
```diff
@@ -221,9 +221,10 @@ void gnrc_sixlowpan_nd_rtr_sol_reschedule(gnrc_ipv6_nc_t *nce, uint32_t sec_dela
 {
     assert(nce != NULL);
     assert(sec_delay != 0U);
+    gnrc_ipv6_netif_t *iface = gnrc_ipv6_netif_get(nce->iface);
     vtimer_remove(&nce->rtr_sol_timer);
     vtimer_set_msg(&nce->rtr_sol_timer, timex_set(sec_delay, 0), gnrc_ipv6_pid,
-                   GNRC_SIXLOWPAN_ND_MSG_MC_RTR_SOL, nce);
+                   GNRC_SIXLOWPAN_ND_MSG_MC_RTR_SOL, iface);
```
Comment (authmillenon) [inline sys/net/gnrc/network_layer/sixlowpan/nd/gnrc_sixlowpan_nd.c]: The function is wrong. The timer is in the nce, it is supposed to be send to the neighbor (represented by the nce) so the function needs to get the nce, not the interface.

Comment (OlegHahm) [inline sys/net/gnrc/network_layer/sixlowpan/nd/gnrc_sixlowpan_nd.c]: So, _rtr_sol_reschedule() and gnrc_sixlowpan_nd_mc_rtr_sol() need to be changed?

Comment (authmillenon) [inline sys/net/gnrc/network_layer/sixlowpan/nd/gnrc_sixlowpan_nd.c]: Ah sorry... You are right… I thought it was changing `GNRC_SIXLOWPAN_ND_MSG_MC_RTR_SOL`. But then the timer needs to be changed to `iface->rtr_sol_timer`, too.

**Comment by OlegHahm** [comment]
addressed

**Comment by authmillenon** [comment]
ACK, please squash.

**Comment by OlegHahm** [comment]
squashed

**Pull request merged**